<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopCameraRenderWdg</class>
 <widget class="QWidget" name="DesktopCameraRenderWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>摄像头</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DesktopCameraRenderWdg{
background-color: rgb(50, 50, 50,1);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_top_left" native="true">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_top" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#widget_top{
background-color: rgb(50, 50, 50);
}
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_top_right" native="true">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_left" native="true">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_center" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(50, 50, 50);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>49</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_closeCamera">
                <property name="minimumSize">
                 <size>
                  <width>112</width>
                  <height>112</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>112</width>
                  <height>112</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/vhallActive/img/vhallactive/camera_close.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>49</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(50, 50, 50);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_no_start">
        <property name="styleSheet">
         <string notr="true">
border-image: url(:/vhallActive/img/vhallactive/live_no_start.png);</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_right" native="true">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_bottom_left" native="true">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_bottom" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_bottom_right" native="true">
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50, 50, 50,1);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
